common/backoff/retrypolicy.go

```diff
@@ -27,9 +27,10 @@ import (
 )
 
 const (
-	// NoInterval represents Maximim interval
-	NoInterval                      = 0
-	done              time.Duration = -1
+	// NoInterval represents Maximum interval
+	NoInterval = 0
+	// Done indicate that there should be no retry
+	Done              time.Duration = -1
```

Comment:
why does this need to be public ?

Reply:
previously i was trying to make the priority queue do rate limiting / back off and retry.

i can change it back